src/ducks/Watchlists/Widgets/Table/CompareInfo/CompareInfo.js

```diff
@@ -1,9 +1,20 @@
 import React from 'react'
+
+import Actions from './Actions/index';
 import styles from './CompareInfo.module.scss'
 
-const CompareInfo = ({ selected, cleanAll }) => {
+const CompareInfo = ({ type, selected, cleanAll, watchlist, refetchAssets }) => {
```

Comment:
What is refetchAssets?

Reply:
Refreshes the wishlist items. We call this method after deleting or adding (undo) wishlist items.